common_features.mk

```diff
@@ -221,7 +221,7 @@ ifneq (,$(filter $(MCU), atmega16u2 atmega32u2))
 endif
     SRC += $(QUANTUM_DIR)/color.c
     SRC += $(QUANTUM_DIR)/rgb_matrix.c
-    ifneq ($(strip $(RGB_MATRIX_DRIVER)), SN32F248B)
+    ifneq ($(strip $(RGB_MATRIX_DRIVER)), SN32F24xx)
```

Comment:
Is the exclusion of $(QUANTUM_DIR)/rgb_matrix_drivers.c for SN32F24xx still needed?
The 26x driver has no issues when rgb_matrix_drivers.c is included in the build, so maybe it's also fine for 24xx.